source/kraken/maintenance_worker.cpp

```diff
@@ -242,7 +239,14 @@ void MaintenanceWorker::listen_rabbitmq(){
     LOG4CPLUS_INFO(logger, "start event loop");
     data_manager.get_data()->is_connected_to_rabbitmq = true;
     while(true){
+        auto now = pt::microsec_clock::universal_time();
+        //We don't want to try to load realtime data every second
+        if(now  > this->next_try_realtime_loading){
+            this->next_try_realtime_loading = now + pt::seconds(conf.kirin_retry_timeout());
+            this->load_realtime();
```

Comment:
```load_realtime``` already check that, and others things, here we just don't want to try a call each second